### ISSUE TITLE
support for inline animated gifs

### ISSUE THREAD

**Issue by S2-** [siacs/Conversations#1939]
It would be nice if animated gifs shared in a conversation would be shown directly inline. at the moment an animated gif get's converted to jpg and shared ad jpg.

**Comment by syntafin** [comment]
You need to deactivate the picture compression in settings :)!

**Comment by timosl** [comment]
Maybe animated .gifs should be excluded from being compressed.

**Comment by tigre-bleu** [comment]
Either exclude GIF or at least add an option to "Keep GIF animation" if picture compression is activated.

**Comment by jkufner** [comment]
Antoine Vacher wrote, on 22.8.2016 11:07:
> Either exclude GIF or at least add an option to "Keep GIF animation" if
> picture compression is activated.

Such option does not make sense. Resizing without preserving animation
is broken unless it is explicitly desired to disable the animation
(avatars in user profiles for example, but that is not the case).

**Comment by blevant** [comment]
If this is inflicted please set default to not-loop animation.

**Comment by blevant** [comment]
I could see this to extend emoji use, but would tire of large gif 
> 99x99 pixels 
> 500kB

**Comment by JohnnyCalavera** [comment]
Since Open Whisper Systems added GIPHY to the Signal Messenger I guess more and more people will be using GIFs in their conversations. 
I'd like to see inline GIF animations in 'conversations' too.

**Comment by licaon-kter** [comment]
Their implementation looks out of this projects scope though: https://whispersystems.org/blog/giphy-experiment/

**Comment by JohnnyCalavera** [comment]
Absolutely. That's why I don't see a gif search engine coming to conversations, but we could have gif animations playing in the chat like they do in Signal for ages.

**Comment by stopANALytics** [comment]
No.  Please, no.

**Comment by afonari** [comment]
Any updates here? I cannot move bunch of people to use conversations because of this!

**Comment by licaon-kter** [comment]
https://medium.com/@giphy/giphy-to-join-facebook-as-part-of-the-instagram-team-e7ea8d32d7b6

**Comment by ver-luisant** [comment]
Any news on this subject? xabber implements this feature natively

**Comment by trymeouteh** [comment]
Would like to see this also. Many messaging apps do this to brings GIFs to life in the conversation.

**Comment by ver-luisant** [comment]
Well, just found this fork of Conversations that does the job nicely, and adds many other improvements :-)
https://codeberg.org/Arne/monocles_chat

**Comment by licaon-kter** [comment]
@ver-luisant that's a fork of Blabber that's a fork of Conversations, _inception style_ :)
